adhocracy-plus/assets/scss/_variables.scss

```diff
@@ -6,6 +6,8 @@ $brand-secondary-dark: #46caad !default; //darker terquoise
 
 $brand-tertiary: #008dc5 !default; // prussian blue
 
+$brand-success: $brand-tertiary; //dark lime green: #0a820a
```

Comment:
you overwrite the brand-success below. So, if we want another colour as brand-success and we do not yet have that colour, you'll have to add it to the variables with another name. Or you just use brand-tertiary, because that's what we want to use? Then you could add u-brand-tertiary to the utilities instead and use that in the html. Or if that is only used for the prgress line, you could also add it as a component